platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -155,6 +159,27 @@ public void run() {
 		return true;
 	}
 
+	public boolean onTouchEvent(final MotionEvent event) {
+		if ((event.getSource() & InputDevice.SOURCE_MOUSE) != InputDevice.SOURCE_MOUSE || event.getAction() != MotionEvent.ACTION_MOVE) {
+			return false;
+		}
+		final int x = Math.round(event.getX());
+		final int y = Math.round(event.getY());
+		for (int i = 0; i < pressedMouseButtons.length; i++) {
+			if(pressedMouseButtons[i]) {
+				final int button = i + 1;
+				queueEvent(new Runnable() {
+					@Override
+					public void run() {
+						GodotLib.mouse_moved_pressed(button, x, y);
+					}
+				});
+			}
+		}
+		return true;
+	}
+
```

Comment:
What's the point of this logic?

You can check for `drag` events by using `getButtonState()` to find out which button are pressed when receiving the `ACTION_HOVER_MOVE` in `onGenericMotionEvents()`.

Reply:
`ACTION_HOVER_MOVE ` event only fire when no button is pressed, when a button us pressed no event fires at all in `onGenericMotionEvents()`